common/exception/src/exception_code.rs

```diff
@@ -145,8 +145,8 @@ build_exceptions! {
 
     // meta store errors
     MetaStoreDamaged(2004),
-    MetaStoreAlreadyExists(2005),
-    MetaStoreNotFound(2006),
+    //MetaStoreAlreadyExists(2005),
+    //MetaStoreNotFound(2006),
 
```

Comment:
There are some sled errors conversion codes in this file, should remove or modify these codes.